pkg/infinispan/cluster.go

```diff
@@ -62,6 +70,8 @@ type ClusterInterface interface {
 	ExistsCache(cacheName, podName string) (bool, error)
 	CreateCacheWithTemplate(cacheName, cacheXML, podName string) error
 	CreateCacheWithTemplateName(cacheName, templateName, podName string) error
+	CreateCache(cacheName, jsonConfig, podName string) error
+	GetCacheConfig(cacheName, podName string) (cacheConfig string, err error)
```

Comment:
We need to make sure that all of the new methods are added to this interface so that we can access via `infinispan.ClusterInterface`.